docs/tutorials/mongoid-documents.txt

```diff
@@ -176,6 +177,31 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
+The StringifiedSymbol Type
+--------------------------
+
```

Comment:
I would add a bit of background explaining that using the `Symbol` field type will cause Mongoid to serialize a field into `BSON::Symbol`s, but that `BSON::Symbol` is deprecated (basically give a bit more background about why someone would want to use this type).